<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>traininfoselect</class>
 <widget class="QWidget" name="traininfoselect">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>876</width>
    <height>413</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_3">
   <item>
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QLineEdit" name="lineEdit"/>
       </item>
       <item>
        <widget class="QPushButton" name="pushButtonFind">
         <property name="styleSheet">
          <string notr="true"/>
         </property>
         <property name="text">
          <string>    查找</string>
         </property>
         <property name="icon">
          <iconset resource="../res.qrc">
           <normaloff>:/image/image/serch.png</normaloff>:/image/image/serch.png</iconset>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="labelMovie">
         <property name="text">
          <string/>
         </property>
         <property name="pixmap">
          <pixmap resource="../res.qrc">:/image/image/refresh.png</pixmap>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pushButtonRefresh">
         <property name="minimumSize">
          <size>
           <width>40</width>
           <height>0</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>40</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true">QPushButton:hover
{
    color:skyblue;
}

QPushButton
{
background-color:transparent;
}</string>
         </property>
         <property name="text">
          <string>刷新</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="horizontalSpacer">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_3">
       <item>
        <widget class="QTableWidget" name="tableWidget">
         <property name="styleSheet">
          <string notr="true"/>
         </property>
         <column>
          <property name="text">
           <string>车次</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>类型</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>起始站</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>终点站</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>发车时间</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>到站时间</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>硬座数</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>卧铺数</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>票价</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>收入总额</string>
          </property>
         </column>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="groupBox">
         <property name="title">
          <string/>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout">
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_2">
            <item>
             <widget class="QWidget" name="widget" native="true">
              <property name="minimumSize">
               <size>
                <width>100</width>
                <height>0</height>
               </size>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QWidget" name="widgetSecond" native="true">
              <property name="minimumSize">
               <size>
                <width>100</width>
                <height>0</height>
               </size>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../res.qrc"/>
 </resources>
 <connections/>
</ui>
